server/api.go

```diff
@@ -480,6 +491,100 @@ func (p *Plugin) getYourPrs(w http.ResponseWriter, r *http.Request) {
 	w.Write(resp)
 }
 
+func (p *Plugin) searchIssues(w http.ResponseWriter, r *http.Request) {
+	config := p.getConfiguration()
```

Comment:
Nit: We're not checking for the correct HTTP method here. It's not being done for other endpoints in the plugin either, so it's probably not a big deal.

Reply:
added method check